docker/build-bas-sitemap.sh

```diff
@@ -3,9 +3,8 @@
 set -e
 
 # 'download' does not work for some reason, but 'public' permissions do
-mc stat minio/sitemaps || mc mb minio/sitemaps
 mc policy set public minio/sitemaps
-
+mc stat minio/sitemaps || mc mb minio/sitemaps
```

Comment:
Thanks for putting that back but it's going to have to go above the line before it, because it is supposed to check whether that directory exists and then create if if it doesn't (and then the policy can be set)